src/vs/workbench/contrib/search/browser/searchView.ts

```diff
@@ -1909,10 +1909,10 @@ export class SearchView extends ViewPane {
 	}
 
 	public override saveState(): void {
-		const patternExcludes = this.inputPatternExcludes.getValue().trim();
-		const patternIncludes = this.inputPatternIncludes.getValue().trim();
-		const onlyOpenEditors = this.inputPatternIncludes.onlySearchInOpenEditors();
-		const useExcludesAndIgnoreFiles = this.inputPatternExcludes.useExcludesAndIgnoreFiles();
+		const patternExcludes = this.inputPatternExcludes ? this.inputPatternExcludes.getValue().trim() : '';
+		const patternIncludes = this.inputPatternIncludes ? this.inputPatternIncludes.getValue().trim() : '';
+		const onlyOpenEditors = this.inputPatternIncludes ? this.inputPatternIncludes.onlySearchInOpenEditors() : false;
+		const useExcludesAndIgnoreFiles = this.inputPatternExcludes ? this.inputPatternExcludes.useExcludesAndIgnoreFiles() : true;
```

Comment:
This should be the same:

```suggestion
		const patternExcludes = this.inputPatternExcludes?..getValue().trim() ?? '';
		const patternIncludes = this.inputPatternIncludes?.getValue().trim() ?? '';
		const onlyOpenEditors = this.inputPatternIncludes?.onlySearchInOpenEditors() ?? false;
		const useExcludesAndIgnoreFiles = this.inputPatternExcludes?.useExcludesAndIgnoreFiles() ?? true;
```